config.json

```diff
@@ -49,7 +49,7 @@
         "bootstrap": ["localhost"]
     },
     "vaultd": {
-        "host": "s3-vault.service.strato",
+        "host": "localhost",
```

Comment:
mmm u have to put here a service-gw port, no?

Reply:
Next line says:
        "port": 8600